crates/bevy_picking_input/src/mouse.rs

```diff
@@ -51,14 +51,16 @@ pub fn mouse_pick_events(
             MouseButton::Right => PointerButton::Secondary,
             MouseButton::Middle => PointerButton::Middle,
             MouseButton::Other(_) => continue,
+            MouseButton::Back => continue,
```

Comment:
Considering it's just used as part of that debug UI (the `[ ] [ ] [ ]`) and the `Other` was not being handled, I'd assume there's no use case.

Otherwise, this struct and its implementation would likely have to be extended.

https://github.com/aevyrie/bevy_mod_picking/blob/1471a9f83435f3fdd43829b90391e6ab965e0e73/crates/bevy_picking_core/src/pointer.rs#L101-L130

Reply:
So, after further review, I found out I was wrong. However, I'm not sure if `PointerPress.is_any_pressed` should include `MouseButton::Back` and `MouseButton::Forward` or not, it could break expectations for anyone using it.

And if we are exposing `MouseButton::Forward` and `MouseButton::Backward` in `PointerPress`, we should also expose `MouseButton::Custom(_)` to offer optional but full flexibility.

Thoughts?

Reply:
We'd add `back` and `forward` fields to `PointerPress` to match the new events. In terms of the debug UI, I agree that using the side buttons on the mouse is probably not "useful" or common, but I'm not sure that we need to implement them on the debug UI in order to still allow users to implement events on them (something I could see being more useful).